README.md

```diff
@@ -1 +1,59 @@
 # sygaldry
+
+## Building Sygaldry
+```bash
+go build -o sygaldry
+```
+
+## Dependencies
+```bash
+go get -u -v github.com/smallfish/simpleyaml
```

Comment:
I think we should use "deps" for dependencies so we can have a file that docs and keeps up with our dependencies

Reply:
[Dis wun](https://www.freecodecamp.org/news/an-intro-to-dep-how-to-manage-your-golang-project-dependencies-7b07d84e7ba5/)?

Reply:
use go mods instead (: